### ISSUE TITLE
Delete Django default permissions

### ISSUE THREAD

**Issue by normanjaeckel** [OpenSlides/OpenSlides#1643]

**Comment by ostcar** [comment]
The old code was:

```python
def delete_default_permissions(**kwargs):
    """
    Deletes the permissions, django creates by default for the admin.
    """
    # TODO: Find a way not to create the permissions in the first place.
    #       Meta.default_permissions does not work, because django will
    #       nevertheless create permissions for its own models like "group"
    for p in Permission.objects.all():
        if (p.codename.startswith('add') or
                p.codename.startswith('delete') or
                p.codename.startswith('change')):
            p.delete()
```

**Comment by emanuelschuetze** [comment]
Norman can you do it for 2.0b1? Otherwise let milestone on 2.0b2.

**Comment by emanuelschuetze** [comment]
This is very important for 2.0b2.

**Comment by normanjaeckel** [comment]
#1746 solves some cases but some are still open. See there.

**Comment by emanuelschuetze** [comment]
So still open here (see #1764): Delete the permissions of Django apps auth, contentypes and session.
We should fix this for 2.0b2. @normanjaeckel can you make a pull request for this?
